2.0/gradle/vividus-test-docker.gradle

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2019-2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+buildscript {
+    repositories {
+        gradlePluginPortal()
+    }
+    dependencies {
+        classpath(group: 'com.bmuschko', name: 'gradle-docker-plugin', version: '9.3.4')
+    }
+}
+
+apply plugin: com.bmuschko.gradle.docker.DockerRemoteApiPlugin
+
+tasks.register('copyResources', Copy) {
+    dependsOn = [build, jar]
+    into layout.buildDirectory.dir('docker/libs')
+    from layout.buildDirectory.dir('libs')
+}
+
+tasks.register('createDockerfile', com.bmuschko.gradle.docker.tasks.image.Dockerfile) {
+    dependsOn = [copyResources]
+    doFirst {
+        from("vividus/vividus:${project.ext.vividusVersion}")
+        copyFile('libs', '/vividus/libs/')
+    }
```

Comment:
do we need to define entry point here?

Reply:
<img width="551" alt="image" src="https://github.com/vividus-framework/vividus-build-system/assets/30923336/8f2af11d-47ae-44e6-b3ff-36d085e819b2">
its inherited from parent image